README.md

```diff
@@ -79,7 +79,11 @@ dpkg -i ld-find-code-refs.amd64.deb
 
 #### Windows
 
-A Windows executable of `ld-find-code-refs` is available on the [releases page](https://github.com/launchdarkly/ld-find-code-refs/releases/latest). The following Chocolatey command may be used to install the required dependency, `ag`. If you do not have Chocolatey installed, see `ag`'s documentation for [installation instructions](https://github.com/ggreer/the_silver_searcher#windows).
+**Note**: The underlying dependency used for search, `ag`, has known performance issues when built for Windows. Unofficial Windows builds are available for `ag` from various forks on GitHub, although correctness and performance cannot be guaranteed. If you are experiencing issues running `ld-find-code-refs` on Windows, we recommend using the [Docker image](#docker) instead.
```

Comment:
I noticed that the `README`  doesn't link to the `ag` documentation. Might be worth adding a link, since a web search for `ag` doesn't immediately return useful results.

Reply:
Good call! I updated the docs above the with a link to `ag`'s repository